frontend/public/components/pod-exec.jsx

```diff
@@ -31,6 +31,7 @@ export class PodExec extends React.PureComponent {
     this.onResize = (rows, cols) => this.onResize_(rows, cols);
     this.onData = d => this.onData_(d);
     this.onChangeContainer = index => this.onChangeContainer_(index);
+    this.setFullscreen = fullscreen => this.setFullscreen_(fullscreen);
```

Comment:
Not needed if we use the arrow function in onClick

Reply:
fixed